src/android/app/src/main/res/values-es/strings.xml

```diff
@@ -246,6 +246,8 @@ Se esperan fallos gráficos temporales cuando ésta esté activado.</string>
     <string name="audio_volume">Volumen</string>
     <string name="audio_stretch">Extensión de Audio</string>
     <string name="audio_stretch_description">Extiende el audio para reducir los parones. Cuando se active, la latencia de audio se incrementará y reducirá un poco el rendimiento.</string>
+    <string name="realtime_audio">Activar audio en tiempo real</string>
+    <string name="realtime_audio_description">Simula el audio de la Nintendo 3DS para HLE, corrigiendo el retraso del audio cuando la velocidad es inferior al 100%. Podría causar problemas de desincronización del audio</string>
```

Comment:
Here an updated text for spanish:

Ajusta la velocidad de reproducción de audio para compensar las caídas en la velocidad de emulación de cuadros. Esto significa que el audio se reproducirá a velocidad completa incluso cuando la velocidad de cuadros del juego sea baja. Puede causar problemas de desincronización de audio.